examples/hello/src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import { Fragment } from 'react'
 import { Provider, atom, useAtom } from 'jotai'
```

Comment:
```suggestion
import { atom, useAtom } from 'jotai'
```
Can we remove this too?